internal/cache/db_test.go

```diff
@@ -87,15 +88,13 @@ func TestNew(t *testing.T) {
 				require.NoError(t, err, "Setup: Can't update the file with invalid db content")
 			}
 
-			expiration, err := time.Parse(time.DateOnly, "2004-01-01")
+			if tc.expirationDate == "" {
+				tc.expirationDate = "2004-01-01"
+			}
+			expiration, err := time.Parse(time.DateOnly, tc.expirationDate)
 			require.NoError(t, err, "Setup: could not calculate expiration date for tests")
 
 			cacheOpts := []cache.Option{cache.WithExpirationDate(expiration)}
```

Comment:
nitpick: I would remove the blank line before, and then add a blank line for other options (basically, the option belongs to what we computed above IMHO rather than grouping options)